crates/shared/src/order_validation.rs

```diff
@@ -872,11 +873,12 @@ async fn get_or_create_quote(
 pub fn is_order_outside_market_price(
     sell_amount: &U256,
     buy_amount: &U256,
+    fee_amount: &U256,
```

Comment:
I'm starting to feel like this method should take two structs rather than 6 arguments, but ok